railties/test/application/rake/dev_test.rb

```diff
@@ -0,0 +1,28 @@
+require 'isolation/abstract_unit'
```

Comment:
Don't you have to include `Isolation` (or something like that) to make this work?